src/components/ProjectList/ProjectsCards.jsx

```diff
@@ -27,7 +27,7 @@ const Card = ({
             alt="the framework or language that the project is build upon"
             src={logoLink}
           />
-          <h3 className="Card-Title">{name}</h3>
+          <p style={{fontSize: 20}} className="Card-Title">{name}</p>
```

Comment:
could you use class instead of style here?

Reply:
Sure!